scripts/ft-benchmark-data-sender.py

```diff
@@ -89,24 +85,33 @@ def get_commit() -> tuple[str, datetime]:
 
 
 def commit_to_db(data: dict) -> None:
+    print(data)
     chdir(os.path.expanduser("~/nearcore/benchmarks/continous/db/tool"))
-    with tempfile.NamedTemporaryFile() as fp:
+    with tempfile.NamedTemporaryFile(mode="w", encoding='utf-8') as fp:
         json.dump(data, fp)
-        fp.close()
+        fp.flush()
+        os.fsync(fp.fileno()) 
         subprocess.run(f"cargo run -p cli -- insert-ft-transfer {fp.name}",
                        shell=True)
+        fp.close()
```

Comment:
I think you should be able to remove `flush` and `fsync` by moving this `close` call after `json.dump`.

Reply:
Unfortunately no.  Probably setting some options while creating tmp file object can help, but I spent a lot of time on this function and didn't find such options

Reply:
Error message was a bit unobvious, so I spent a time playing with tmp file options and place of `fp.close()`, before figured out, that it just don't flushing for some reason

Reply:
Can you provide steps to reproduce this? According to Python documentation, `close` is guaranteed to call `flush`: https://docs.python.org/3/library/io.html#io.IOBase.close

Reply:
You need to disable `delete_on_close` though: https://github.com/python/cpython/blob/cde976d85c6632a908dde1ff8695ac5cafd879b6/Lib/tempfile.py#L540